api/v1/serializers/students-serializer.js

```diff
@@ -0,0 +1,143 @@
+
+  rawAccountBalance.currentBalance = parseFloat(rawAccountBalance.currentBalance);
+
+  return new JSONAPISerializer(
+    serializerArgs.resourceType,
+    serializerOptions(serializerArgs),
+  ).serialize(rawAccountBalance);
+};
+
+const serializeAccountTransactions = (rawTransactions, osuID) => {
+  const serializerArgs = getSerializerArgs(osuID, 'AccountTransactionsResult', 'account-transactions', true);
+  const identifierField = osuID;
+
+  _.forEach(rawTransactions, (rawTransaction) => {
+    const rawEntryDate = rawTransaction.entryDate;
+    rawTransaction.amount = parseFloat(rawTransaction.amount);
+    rawTransaction.entryDate = moment.tz(rawEntryDate, 'PST8PDT').utc().format();
+  });
+
+  const rawAccountTransactions = { identifierField, transactions: rawTransactions };
+
+  return new JSONAPISerializer(
+    serializerArgs.resourceType,
+    serializerOptions(serializerArgs),
+  ).serialize(rawAccountTransactions);
+};
+
+const serializeAcademicStatus = (rawAcademicStatus, osuID) => {
+  const serializerArgs = getSerializerArgs(osuID, 'AcademicStatusResult', 'academic-status', false);
+
+  const rawDataByTerm = {};
+  const termGPA = {};
+
+  _.forEach(rawAcademicStatus, (rawRow) => {
+    const rawGPA = {
+      gpa: rawRow.gpa,
+      gpaCreditHours: parseFloat(rawRow.gpaCreditHours),
+      gpaType: rawRow.gpaType,
+      creditHoursAttempted: parseFloat(rawRow.creditHoursAttempted),
+      creditHoursEarned: parseFloat(rawRow.creditHoursEarned),
+      creditHoursPassed: parseFloat(rawRow.creditHoursPassed),
+      level: rawRow.level,
+      qualityPoints: rawRow.qualityPoints,
+    };
+    termGPA[rawRow.term] = _.defaultTo(termGPA[rawRow.term], []).concat(rawGPA);
+  });
+
+  _.forEach(rawAcademicStatus, (rawRow) => {
+    rawDataByTerm[rawRow.term] = {
+      identifierField: `${osuID}-${rawRow.term}`,
+      academicStanding: rawRow.academicStanding,
+      term: rawRow.term,
+      termDescription: rawRow.termDescription,
+      gpa: termGPA[rawRow.term],
+    };
+  });
+
+  const newRawAcademicStatus = [];
+  _.forEach(rawDataByTerm, (rawData) => {
+    newRawAcademicStatus.push(rawData);
+  });
+
+  return new JSONAPISerializer(
+    serializerArgs.resourceType,
+    serializerOptions(serializerArgs),
+  ).serialize(newRawAcademicStatus);
+};
+
+const serializeClassification = (rawclassification, osuID) => {
+  const serializerArgs = getSerializerArgs(osuID, 'ClassificationResult', 'classification', true);
+
+  return new JSONAPISerializer(
+    serializerArgs.resourceType,
+    serializerOptions(serializerArgs),
+  ).serialize(rawclassification);
+};
+
+const serializeGrades = (rawGrades, osuID) => {
+  const serializerArgs = getSerializerArgs(osuID, 'GradesResult', 'grades', false);
+
+  _.forEach(rawGrades, (rawGrade) => {
+    rawGrade.creditHours = parseFloat(rawGrade.creditHours);
+  });
+
+  return new JSONAPISerializer(
+    serializerArgs.resourceType,
+    serializerOptions(serializerArgs),
```

Comment:
Since it seems like `JSONAPISerializer()` is always called with the same parameters, could it be refactored into its own simpler method?